### PR TITLE
PAT-1399 'View in Pathfinder' - Link not present in DPS profile header

### PR DESCRIPTION
**Pull request by steverendell**
Some extra PF roles

### CODE REVIEW

File: app/components/Bookings/Details/header.test.js [hunk 1]
```diff
@@ -87,6 +87,8 @@ describe('Header component', () => {
               categorisationUrl="http://catTool"
               isUseOfForce={false}
               useOfForceUrl="http://useofforce"
+              isPathfinderUser={false}
+              pathfinderUrl=""
```
Comment (steverendell) [inline app/components/Bookings/Details/header.test.js:91]: cleanup test warning

**Comment by JamieDraperUK** [comment]
There seems to be a missing test for the new roles?

File: app/containers/Authentication/tests/reducer.test.js [hunk 2]
```diff
@@ -87,168 +37,42 @@ describe('Authentication reducer', () => {
...
+    ${'SOME_OTHER_ROLE'}             | ${'Cannot view probation documents'} | ${'canViewProbationDocuments'} | ${false}
+    ${'PRISON'}                      | ${'View probation documents'}        | ${'isPrisonUser'}              | ${true}
+    ${'LICENCE_CA'}                  | ${'Licence Case Admin'}              | ${'isLicenceUser'}             | ${true}
+    ${'LICENCE_DM'}                  | ${'Licence Decision Maker'}          | ${'isLicenceUser'}             | ${true}
+    ${'LICENCE_VARY'}                | ${'Vary Licence role'}               | ${'isLicenceUser'}             | ${true}
+    ${'NOMIS_BATCHLOAD'}             | ${'Licence Administrator'}           | ${'isLicenceUser'}             | ${true}
+    ${'LICENCE_READONLY'}            | ${'Read-only Licence Access'}        | ${'isLicenceUser'}             | ${true}
+    ${'NOT_A_LICENCE_ROLE'}          | ${'Not a licence role'}              | ${'isLicenceUser'}             | ${false}
+    ${'PF_PSYCHOLOGIST'}             | ${'Pathfinder Psychologist'}         | ${'isPathfinderUser'}          | ${true}
```
Comment (steverendell) [inline app/containers/Authentication/tests/reducer.test.js:62]: here is the new test ^ !

File: app/containers/Authentication/tests/reducer.test.js [hunk 3]
```diff
@@ -87,168 +37,42 @@ describe('Authentication reducer', () => {
...
+    ${'OTHER'}                       | ${'Some'}                            | ${'isPathfinderUser'}          | ${false}
+  `('For role $role - $description, $flag should be $hasAccess', ({ role, description, flag, hasAccess }) => {
+    const user = {
+      ...userData,
+      accessRoles: [{ roleCode: role, roleDescription: description }],
+    }
+    const state = authenticationReducer(Map({}), userMe({ user }))
+    const userState = state.get('user')
+
+    expect(userState[flag]).toBe(hasAccess)
```
Comment (steverendell) [inline app/containers/Authentication/tests/reducer.test.js:76]: generalised this slightly to allow it to be used by almost all the tests

**Review by JamieDraperUK — APPROVED** [review]
:+1:

**Pull request merged**